src/tox_lsr/test_scripts/runcollection.sh

```diff
@@ -43,6 +43,9 @@ tox --workdir "$TOXINIDIR/.tox" -e "$testlist" 2>&1 | tee "$MY_LSR_TOX_ENV_DIR"/
 
 rval=0
 if [ "${LSR_ROLE2COLL_RUN_ANSIBLE_TESTS:-}" = "true" ]; then
+    # ansible-test needs the modules and meta data
+    pip install pycodestyle pylint
```

Comment:
I think we should move these into tox-default.ini - `testenv:collection` as `deps` - I think it is fine to have to install these even if not desired - it is better to have all the deps in one place